workbench-core/datasets/src/schemas/createDataSet.ts

```diff
@@ -12,10 +12,18 @@ const CreateDataSetSchema: Schema = {
   properties: {
     datasetName: { type: 'string' },
     path: { type: 'string' },
-    region: { type: 'string' }
+    description: { type: 'string' },
+    customMetadata: {
+      type: 'object',
+      properties: {
+        owningProjectId: { type: 'string' }
+      },
+      additionalProperties: false,
+      required: ['owningProjectId']
+    }
   },
   additionalProperties: false,
-  required: ['datasetName', 'path', 'region']
+  required: ['datasetName', 'path', 'customMetadata']
```

Comment:
`customMetadata` should not be required. If a user doesn't have custom metadata to pass in, they shouldnt be required to pass in an empty object

Reply:
But we still have owning project here. Possibly I'd need to move project under main entity